docs/kyma/docs/042-gardener-kyma-deployment.md

```diff
@@ -0,0 +1,80 @@
+
+## Configure your Gardener project
+
+To configure your Gardener project, access the Gardener dashboard and make sure the following conditions are satisfied:
+  - Microsoft Azure Cloud secret is present in the `secrets` tab
+  - account service is added in the `members` tab
+
+>**NOTE:** The Kyma installer does not support clusters on AWS infrastructure as the provider does not support static IP assignment during ELB creation.
+
+## Prepare `Shoot` cluster
+
+**1. Generate a TLS certificate**
+
+`Shoot` clusters require a TLS certificate to run Kyma properly. To generate a certificate for your website’s domain, use one of the Certificate Authorities (e.g. Let's Encrypt).
+
+**2. Fill in the `shoot.yaml.tpl` template.**
+
+The deployment of a Gardener cluster requires data specified in the `shoot.yaml` file. Copy the [shoot.yaml.tpl](../../../installation/resources/gardener/shoot.yaml.tpl) template, rename it to `shoot.yaml`, and fill in the placeholder values:
+
+- `__CLUSTER_NAME__` for the desired name of your cluster
+- `__PROJECT_NAME__` for the name of your project
+- `__PURPOSE__` for the purpose of the cluster (development, evaluation or production)
+- `__AZURE_SECRET_NAME__` for the name of your Azure secret specified it the `secrets` tab in the Gardener dashboard
+- `__TLS_CERT__` for the TLS certificate generated in the first step
+- `__DOMAIN__` for the domain that you control
+
+**3. Deploy your `Shoot` cluster.**
+
+To deploy your `Shoot` cluster, connect to the `Garden` cluster first. Go to the `Members` tab in the Gardener dashboard, find the `Service Account` section, and download the garden cluster's kubeconfig. Establish the connection by exporting the `KUBECONFIG` environment variable:
+```
+export KUBECONFIG={PATH_TO_GARDEN_CLUSTER_KUBECONFIG_FILE}
+```
+
+Make sure you are connected to the right cluster:
+```
+kubectl cluster-info
+```
+
+To deploy your `Shoot` cluster, run this command:
+```
+kubectl create -f installation/resources/gardener/shoot.yaml
+```
+Go to the `Clusters` tab in the Gardener dashboard to track the deployment process. Wait until your cluster is ready. 
+
+To connect to the `Shoot` cluster, choose it from the list and download the kubeconfig file. Establish the connection by exporting the `KUBECONFIG` environment variable:
+```
+export KUBECONFIG={PATH_TO_SHOOT_CLUSTER_KUBECONFIG_FILE}
+```
+
+**4. Add the `managed-standard` Storage Class**
+
+To add the Storage Class, run this command:
+```
+kubectl apply -f installation/resources/gardener/managed-standard.yaml
+```
+
+## Install Kyma
+
+To install Kyma on your `Shoot` cluster, follow the steps provided in the [032-gs-cluster-installation.md](./032-gs-cluster-installation.md) file.
+
+>**NOTE:** To enable interaction with `Azure Broker`, perform the below actions before applying the Custom Resource file (step 6 of the instruction).
```

Comment:
This instruction should be added to the cluster installation document as an optional step valid only when installing on a Gardener cluster. Insert this step in the correct (from the chronological point of view) place in the instruction.